include/material/mohr_coulomb.tcc

```diff
@@ -1,7 +1,7 @@
 //! Constructor with id and material properties
 template <unsigned Tdim>
 mpm::MohrCoulomb<Tdim>::MohrCoulomb(unsigned id,
-                                    const Json& material_properties)
+                                    const Json &material_properties)
```

Comment:
Please run `clang-format` `&` should be with the type so `Json&`

Reply:
Sorry, I think I have run the clang-format.

Reply:
May be there are some thing error in my .emacs file